docs/src/devdocs/dev_guide.md

```diff
@@ -16,13 +16,13 @@ continuing stability of releases. Before making a release one should
    issues can arise that do not lead to actual errors in the doc CI).
 4. Release via the [registration
    issue](https://github.com/SciML/Catalyst.jl/issues/127) with the
-   command:
+   command:  
    
-    ```
-    @JuliaRegistrator register branch=release-15.0.0
-    ```
+   ```
+   @JuliaRegistrator register branch=release-15.0.0
+   ```
```

Comment:
Maybe use a raw block?

https://documenter.juliadocs.org/stable/man/syntax/#@raw-format-block

Reply:
nope. trying just a `@JuliaRegistrator register branch=release-15.0.0` and checking how that looks